card/styles.css

```diff
@@ -97,3 +130,11 @@
 .card-animated i {
   font-size: 2rem;
 }
+
+/* This style for iframe(not working properly) */
+.iframe-style {
+  height: 500px;
+  background-color: transparent;
+  padding-left: 0px;
+  padding-right: 0px;
+}
```

Comment:
Save this as utility as you'll be using it in multiple components

Reply:
Moved to general.css Thank you for the feedback